### PR TITLE
Add the vue standard linter to the eslint config

### LINKED ISSUE #1: Add standard linting
- [ ] Add [vue standard config](https://www.npmjs.com/package/@vue/eslint-config-standard) for eslint
- [ ] Lint the project and fix all the errors

This makes it easier for collaborators, yay 🎐

### PR DESCRIPTION
**Pull request by blascsi**
I have added the @vue/eslint-config-standard to the extends in the .eslintrc.js.
`npm run lint` found no lint errors after this, so I made no changes to the components.

Hope this helps!

Resolves #1

### CODE REVIEW

**Comment by Fohlen** [comment]
Hey @blascsi 
You PR looks good!

I haven't set up Travis so far, so I need to test the linting when I use `dashboard` this weekend / next week.
So, it'll take a few days, no worries, your PR won't stal.

All the best, greetings from Tuebingen to lovely Budapest 🍺

**Comment by blascsi** [comment]
Great, thank you @Fohlen !
Hope everything will be fine :)

**Comment by blascsi** [comment]
Hi @Fohlen 
Since then I have found this [article](https://eslint.vuejs.org/user-guide/#usage) which indicated that the way I added the linter was incorrect. I have corrected the error, now the linter has found errors, and autofixed most of them :)

File: src/components/modals/location/Delete.vue [hunk 1]
```diff
@@ -27,7 +54,8 @@ export default {
   name: "DeleteLocationModal",
   props: {
     name: {
-      type: String
+      type: String,
+      default: '',
```
Comment (blascsi) [inline src/components/modals/location/Delete.vue]: Please check this line, the linter required a default value here, but I wasn't sure what to set (especially in german :smile: )

File: src/components/modals/source/Delete.vue [hunk 2]
```diff
@@ -27,7 +54,8 @@ export default {
   name: "Delete",
   props: {
     url: {
-      type: String
+      type: String,
+      default: '',
```
Comment (blascsi) [inline src/components/modals/source/Delete.vue]: The same here as above.

File: src/components/modals/location/Delete.vue (hunk 1 above)
Comment (Fohlen) [reply src/components/modals/location/Delete.vue]: I believe this can be solved by removing the `default` and replacing it with `required: true`

**Review by Fohlen — CHANGES_REQUESTED** [review]

**Comment by Fohlen** [comment]
Hey @blascsi how are you invoking the linter? Using `npm run lint` will not issue any `linter` errors on the mentioned passage.

**Comment by blascsi** [comment]
I changed the default values to required :smile: Sorry this is my first Vue PR!
And there shouldn't be any lint errors, the linter was autofixing most of them, only the two errors where I made the comments were fixed by hand.

**Comment by Fohlen** [comment]
This PR looks good now! Will merge. Thanks for contributing ✨ 

You can add yourself to the [list of collaborators](https://github.com/eventzimmer/schema/blob/master/CONTRIBUTORS.md), as you like.

Greetings from Vienna!

**Comment by blascsi** [comment]
Awesome! Thank you and greetings from Budapest! Have a great day!

**Pull request merged**